CHANGES.txt

```diff
@@ -4,8 +4,11 @@ Changes
 2.6 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- compatibility for python 3
+- update code to work with python2 and python3
+- update doctests to work with python2 and python3
+- adjust egg to work with newer version of setuptools
+- reimplment ``zope.interface.implements`` with a grokker
```

Comment:
Also: this reimplementation is active only if you use grokcore.component on Python 3.  This is an important detail that should be mentioned in the changelog.